docs/pages/2019_MARVEL_Psik_MaX/sections/verdi_cmdline.rst

```diff
@@ -39,406 +54,385 @@ The first output line should look like
-relaxation of a BaTiO\ :sub:`3` unit cell run with Quantum Espresso ``pw.x``.
-You can check the information on this node and get the PK with:
+Again, since the UUID is unchanging we know what it represents: a Quantum Espresso ``pw.x`` relaxation of a BaTiO\ :sub:`3` unit cell.
 
 .. code:: console
 
     Property       Value
     -------------  ------------------------------------
     type           CalcJobNode
-    pk             17
+    pk             828
     uuid           ce81c420-7751-48f6-af8e-eb7c6a30cec3
     label
     description
-    ctime          2019-04-18 10:51:16.710803+00:00
-    mtime          2019-04-18 10:51:51.012617+00:00
+    ctime          2014-10-27 17:51:21.781045+00:00
+    mtime          2019-05-09 14:10:09.307986+00:00
     process state  Finished
     exit status    0
-    computer       [1] localhost-direct
-
-    Called by      PK  Type
-    -----------  ----  -------------
-    CALL_CALC       4  WorkChainNode
+    computer       [1] daint
 
-    Inputs         PK    Type
-    -------------  ----  -------------
+    Inputs      PK    Type
+    ----------  ----  -------------
     pseudos
-        Ba         26    UpfData
-        O          27    UpfData
-        Ti         1     UpfData
-    code           10    Code
-    kpoints        16    KpointsData
-    parameters     15    Dict
-    parent_folder  9     RemoteData
-    settings       14    Dict
-    structure      3     StructureData
-
-    Outputs              PK  Type
-    -----------------  ----  --------------
-    output_band          23  BandsData
-    output_parameters    22  Dict
-    output_structure     20  StructureData
-    output_trajectory    21  TrajectoryData
-    remote_folder        18  RemoteData
-    retrieved            19  FolderData
-
-*Keep in mind that you can also use just a part (beginning) of the UUID,
-as long as it is unique, to show the node information information.* For
-example, to display the above information, you could also type
-``verdi node show ce81c420``. In what follows, we are going to
-mention only the prefixes of the UUIDs since they are sufficient to
-identify the correct node.
+        Ba      611   UpfData
+        O       661   UpfData
+        Ti      989   UpfData
+    code        825   Code
+    kpoints     811   KpointsData
+    parameters  829   Dict
+    settings    813   Dict
+    structure   27    StructureData
+
+    Outputs                    PK  Type
+    -----------------------  ----  -------------
+    output_kpoints           1894  KpointsData
+    output_parameters          62  Dict
+    output_structure           61  StructureData
+    output_trajectory_array    63  ArrayData
+    remote_folder             357  RemoteData
+    retrieved                  60  FolderData
+
+The output should show a overview with some general information about the process.
+Additionally, it will show what its inputs and outputs were and if it was called by another process, or it itself called other processes.
```

Comment:
```suggestion
Additionally, it will show what its inputs and outputs were and if it was called by another process, or if it called other processes itself.
```

Reply:
Done